tests/modules/RouterTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ *
+ * RouterTest.php is unit test file for Router module (modules/Router.php).
+ *
+ */
+namespace Tests\FirstAide;
+
+require_once str_replace('tests/modules', '', dirname(__FILE__)).'/includes/constants.php';
+
+/**
+ * Importing required modules/wrappers
+ */
+use FirstAide\User;
+use FirstAide\Router;
+use FirstAide\MysqlDatabase;
+use PHPUnit\Framework\TestCase;
+use Tests\MysqlWrapperMockTrait;
+
+class RouterTest extends TestCase
+{
+    /**
+     * Importing mock MySQL wrapper
+     */
+    use MysqlWrapperMockTrait;
+
+    /**
+     * @dataProvider mysqlMockProvider
+     */
+    public function testGetPage($databaseMock)
```

Comment:
Refactoring by creating model is a longer task that may need changes across the app. 

For this sprint, lets get the methods created for the cases like we discussed and reduce the size of getPage method.